RESEARCH_Keys-to-the-Kingdom-The-Morrow-Days-and-Demesnes-2026-06-04.md

```
@@ -0,0 +1,80 @@
+authority: LOGAN
+status: reference
+type: research-report
+topic: Keys to the Kingdom — the House, the Morrow Days, and their demesnes
+verified-by: Claude Code (web research 2026-06-04)
+date: 2026-06-04
+---
+
+# The House — The Morrow Days and Their Demesnes
+
+> A standalone reference on **Garth Nix's *Keys to the Kingdom*** — the **House**, its seven **Trustees** (the *Morrow Days*), and the seven **demesnes** they rule. Companion to the ASOIAF `[[RESEARCH_A-Song-of-Ice-and-Fire-The-Small-Council-2026-06-03]]`; the **syncretic crosswalk** between the two systems lives **only** in `[[THE-SEVENFOLD-BODY-SEATS-TRUSTEES-GEMS-2026-06-03]]`. This node holds **no cross-mapping** — pure source.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)